pkg/issuer/acme/dns/azuredns/azuredns.go

```diff
@@ -205,7 +206,7 @@ func (c *DNSProvider) getHostedZoneName(fqdn string) (string, error) {
 	_, err = c.zoneClient.Get(context.TODO(), c.resourceGroupName, util.UnFqdn(z), nil)
 
 	if err != nil {
-		return "", fmt.Errorf("Zone %s not found in AzureDNS for domain %s. Err: %v", z, fqdn, err)
+		return "", fmt.Errorf("Zone %s not found in AzureDNS for domain %s. Err: %v", z, fqdn, stabilizeError(err))
```

Comment:
Looking back at my review comment: https://github.com/cert-manager/cert-manager/pull/5663/files#r1060724707 I liked that @weisdd had logged the original error which while returning the redacted error, so that the redacted information is at least available in the logs for easier debugging (for people who have permission to view the cert-manager logs).

Reply:
I'll add a a log statement here.